package.json

```diff
@@ -56,7 +57,7 @@
     "terser-webpack-plugin": "5.1.2",
     "ts-jest": "^26.1.4",
     "ts-loader": "^6.2.2",
-    "typedoc": "0.20.36",
+    "typedoc": "0.19.2",
```

Comment:
Any particular reason for this typedoc downgrade?

Reply:
For some reason the `0.20.x` version doesn't work to build the AvalancheJS deps so I downgraded

```zsh
Info: Loaded plugin typedoc-plugin-external-module-name
Error: Unknown option: --mode
Error: Unknown option: --mode
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! avalanche@3.6.1 docshtml: `npx typedoc --plugin typedoc-plugin-external-module-name --mode file src`
npm ERR! Exit status 1
```